File: infrastructure-servers/discovery-server/src/main/java/ua/piraeusbank/banking/discovery/DiscoveryServer.java
package ua.piraeusbank.banking.discovery;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer;
import org.springframework.context.annotation.Import;

@SpringBootApplication
@EnableEurekaServer
@Import(BrokerServerConfig.class)
public class DiscoveryServer {

	public static void main(String[] args) {
		SpringApplication.run(DiscoveryServer.class, args);
	}
}

File: infrastructure-servers/discovery-server/src/main/java/ua/piraeusbank/banking/discovery/DiscoveryServer.kt
package ua.piraeusbank.banking.discovery

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer

@SpringBootApplication
@EnableEurekaServer
class DiscoveryServer

fun main(args: Array<String>) {
    SpringApplication.run(DiscoveryServer::class.java, *args)
}